cloudflare/src/api/rightsrequests.js

```diff
@@ -72,9 +77,15 @@ export async function rightsRequestsApi(request, env) {
   }
 
   // Review routes (reviewer perspective)
+  if (request.method === 'GET' && path.endsWith('/rightsrequests/reviews/reviewers')) {
+    return listAvailableReviewers(request, env);
+  }
   if (request.method === 'GET' && path.endsWith('/rightsrequests/reviews')) {
     return listReviewsForReviewer(request, env);
   }
+  if (request.method === 'POST' && path.endsWith('/rightsrequests/reviews/assign-to')) {
+    return assignReviewToReviewer(request, env);
```

Comment:
could we use assign for both? assign a review to a user?
don't think we need 2 as both have same intent of assignin a review to a user